scripts/ProcessTemplates.ps1

```diff
@@ -118,8 +118,9 @@ Function AddCategory() {
     $categories.$categoryName.DescriptionByLanguage = @{ }
 
     $categorySettings | Get-Member -type NoteProperty | Foreach-Object {
-        # ignore the $schema field which is needed for localization of categoryResources.json
-        if ($_.name -ne '$schema') {
+        # only process languages field in categoryResources.json

```

Comment:
not being an expert at powershell, but doesn't this now loop over every property in the file and then n^2 check to see if it is in supported languages?  couldn't you just loop over all the supported languages and see if there's a property with that name, one loop instead of 2 nested ones?  it isn't *that* big of a deal since its just a couple properties and 17 languages, but still seems like extra work this way?

Reply:
since we know what the lang is we should just match that and eliminate the loops